torch/csrc/jit/codegen/cuda/root_domain_map.h

```diff
@@ -397,6 +397,10 @@ class TORCH_CUDA_CU_API ComputeAtRootDomainMapBuilder
     mapPointwiseOrReductionOp(op);
   }
 
+  void handle(ViewDtypeOp* op) override {
+    mapPointwiseOrReductionOp(op);
```

Comment:
This will be important to fill out manually when modifying the rfactor domain of the producer/ root domain of the output..